x-pack/plugins/endpoint/common/types.ts

```diff
@@ -236,6 +236,10 @@ interface AlertMetadata {
   // Alert Details Pagination
   next: string | null;
   prev: string | null;
+
+  state?: {
+    active: boolean;
```

Comment:
what is this for?

Reply:
the alert details state, this will change to match more of marshall's pr's structure once he merges